packages/codegen/example/build-example.mjs

```diff
@@ -31,11 +31,11 @@ async function main() {
 }
 
 function replaceImport(source) {
-  return source.replaceAll('@xata.io/client', '../../client/src');
+  return source?.replaceAll('@xata.io/client', '../../client/src');
```

Comment:
Is source possible undefined?

Reply:
Yes, if the file doesn't exist previously